package exception_handling;

public class Throws {
    private static void trywithResources() {
        Connection conn;
        ResultSet rs;

        try {
            conn
        }
    }
}
